### ISSUE TITLE
Fatal exception of logging

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#8761]
While playing around a bit with the new LoggingManager I constantly receive the following error already on the first step:

```
2020-08-06 22:36:24.092 31987-31987/cgeo.geocaching E/AndroidRuntime: FATAL EXCEPTION: main
    Process: cgeo.geocaching, PID: 31987
    java.lang.RuntimeException: Unable to start activity ComponentInfo{cgeo.geocaching/cgeo.geocaching.log.LogCacheActivity}: java.lang.IllegalArgumentException: Object returned from onCreateLoader must not be null
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
     Caused by: java.lang.IllegalArgumentException: Object returned from onCreateLoader must not be null
        at androidx.loader.app.LoaderManagerImpl.createAndInstallLoader(LoaderManagerImpl.java:385)
        at androidx.loader.app.LoaderManagerImpl.initLoader(LoaderManagerImpl.java:421)
        at cgeo.geocaching.connector.gc.GCLoggingManager.init(GCLoggingManager.java:127)
        at cgeo.geocaching.log.LogCacheActivity.onCreate(LogCacheActivity.java:287)
        at android.app.Activity.performCreate(Activity.java:7802)
        at android.app.Activity.performCreate(Activity.java:7791)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1299)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7356) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930) 
```

The cache is stored offline already. Error occurs directly when pressing the "log" button. The log dialog does not even show up, c:geo directly crashes (or at least the logging activity does) - returns to cache list.
Probably due to no GC login info having been stored, because when I added login data for this connector, the dialog showed up. - But still: In this case an error message should be shown instead of crashing the activity.

@eddiemuc

**Comment by eddiemuc** [comment]
@moving-bits you are working late
I am looking into this

**Comment by eddiemuc** [comment]
@moving-bits, @Lineflyer on another note, I will be on vacation the next week and thus not really connected. I probaly won't be able to do anything on c:geo

**Comment by eddiemuc** [comment]
added PR #8762 to fix this. Build seems to hang on unrelated test case, can you please check?

**Comment by fm-sys** [comment]
> added PR #8762 to fix this. Build seems to hang on unrelated test case, can you please check?

This is a known bug with exists since some months. Just do some retests, until the tests will finish without an exemption

**Comment by moving-bits** [comment]
BTW: Can you add the marker `(fix #xxxx)` or `(fixes #xxxx)` to your commit message, with `xxxx` being the issue number which is to be fixed by your PR. Then CI can/will automatically close the issue being referred to on merging the PR to master branch. Thanks.

**Comment by moving-bits** [comment]
fixed with #8762

**Comment by eddiemuc** [comment]
> BTW: Can you add the marker `(fix #xxxx)` or `(fixes #xxxx)` to your commit message, with `xxxx` being the issue number which is to be fixed by your PR. 

Will do that in the Future. Thanks for the hint
